.gitignore

```diff
@@ -1,5 +1,7 @@
-/target/
+target/
 !.mvn/wrapper/maven-wrapper.jar
+*.class
+*application.properties
```

Comment:
I have my crawler `application.properties` file untracked like above because it contains the secret keys (but I have another `application.properties`  file with different information in the api).
A separate file may be a better solution?